docs/lake/README.md

```diff
@@ -374,8 +383,8 @@ These range queries are efficiently implemented as the data is laid out
 according to the pool key and seek indexes keyed by the pool key
 are computed for each data object.
 
-Lake queries also can refer to HEAD either implicitly by omitting
-the `from` operator:
+Lake queries also can refer to HEAD (i.e., the branch context set in the most
+recent `use` command) either implicitly by omitting the `from` operator:
```

Comment:
Is this an accurate thing to say? As I was going through the current README I noticed that while `-HEAD` is gone as an option, there's still plenty of "HEAD" references remaining in the doc. The first time the word "HEAD" appears in the doc (in the "Use" section) it's mentioned without any prior context. Once again, users accustomed to the Git concepts will probably understand what this is saying, but then I get concerned that other users might feel alienated. Maybe a separate section about the role of "HEAD" would be justified in the doc. For now, I'm hoping what I'm written here can stand on its own, since it's how I happen to look at it as a user: Once I've done `zed lake use`, I'm free to run all my `query` and `load` commands and know that was my context without having to specify `from` in my queries or reference a pool in my `load`.

Reply:
Yes looks good.  HEAD is a concept that shows up in the log listing and is how git refers to this concept, but we can revisit it.  It's really useful when you run zed lake log to see where HEAD is at relative to other branch references.